test/integration_helper.js

```diff
@@ -15,7 +15,27 @@ const testHandlesNullData = (stubName, func, ...args) => {
   })
 }
 
+const testTimeBound = (metric, slug, from, to, currency, days, numberOfDays, formatDate) => {
+  const timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
```

Comment:
you're redefining the supported time bounds in a few places - maybe unify with an import?

Reply:
Because in ECMA 5 there isn't a 'require' for importing, thus I cannot import stuff in there. I can make the timeBoundVariable in setup.js, but it's pointless if it is just for the integration_helper